src/Activity.js

```diff
@@ -1,141 +1,103 @@
-class Activity { //should probably be renamed
-  constructor(activityData) { //constructor sucks but has to stay
-    this.activityData = activityData //
+class Activity {
+  constructor(activityData) {
+    this.activityData = activityData
   }
-  //// begin unused methods
-  getMilesByDate(id, date, user = undefined, userRepo) {  //returns miles walked on a specific date
-    // THIS IS NEVER USED needs to be displayed on page
-    let userStepsByDate = this.getUserDataForDay(id, date, 'numSteps').amount;
-    let miles = parseFloat(((userStepsByDate * user.strideLength) / 5280).toFixed(1))
-    return miles || undefined;
```

Comment:
We are using this method in the population of todays data. I dont think we can delete it.